pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -112,6 +112,9 @@ func ValidateClusterConfig(cfg *ClusterConfig) error {
 		if err := validateNg(ng.NodeGroupBase, path); err != nil {
 			return err
 		}
+		if ng.DisableASGTagPropagation != nil {
+			logger.Warning("field DisableASGTagPropagation for nodegroup has been deprecated and has no effect. Please use PropagateASGTags instead for nodegroup %s!", ng.Name)
+		}
```

Comment:
do we have a follow up issue to remove the deprecated field? We can be quicker to remove it than the normal 6 months if you like, given very few ppl will have used it

Reply:
https://github.com/weaveworks/eksctl/issues/5007